src/screens/ShipItemDetails/index.tsx

```diff
@@ -94,6 +101,7 @@ const ShipItemDetails = () => {
   };
 
   const quantityPickedChange = (query: string) => {
+
```

Comment:
Remove this empty line here and add an empty line after this function

Reply:
okay